Exiled.API/Features/Roles/Scp079Role.cs

```diff
@@ -44,11 +44,11 @@ public Camera Camera
         /// </summary>
         public Scp079PlayerScript.Ability079[] Abilities
         {
-            get => Owner.ReferenceHub.scp079PlayerScript?.abilities;
+            get => Script?.abilities;
             set
             {
-                if (Owner.ReferenceHub.scp079PlayerScript is not null)
-                    Owner.ReferenceHub.scp079PlayerScript.abilities = value;
+                if (Script is not null)
```

Comment:
There are a lot of null checks here, I can't understand this
There are no null checks in other ScpRoles.
Only `scp079PlayerScript` can be null?
I do not think so